src/biconnectivity/biconnect.jl

```diff
@@ -31,14 +31,13 @@ function biconnected_components(g::Graph)
             empty!(state.stack)
         end
     end
-
     return state.biconnected_comps
 end
 
 """
 Does a DFS visit and stores the depth and low-points of each vertex
 """
-function visit!(g::Graph, state::Biconnections, u::Int, v::Int)
+function visit!(g::Graph, state::Biconnections, u::Integer, v::Integer)
```

Comment:
That makes sense as a constraint on the top level function. But can we leave this internal function of AbstractGraph? I guess we can always loosen constraints in the future without breaking interfaces so we should let things stay this way.